src/ORM/Connect/DBSchemaManager.php

```diff
@@ -849,6 +861,29 @@ abstract public function IdColumn($asDbValue = false, $hasAutoIncPK = true);
      */
     abstract public function checkAndRepairTable($tableName);
 
+    /**
+     * Determines if we should be checking and repairing tables generally, and whether the passed in table
+     * is on the ignore list.
+     */
+    private function canCheckAndRepairTable(string $tableName): bool
+    {
+        if (!Config::inst()->get(static::class, 'check_and_repair_on_build')) {
+            return false;
+        }
+
+        // Return false if $tableName belongs to any model in the data hierarchy of any class in the ignore list
+        $ignoreDataObjects = Config::inst()->get(static::class, 'exclude_models_from_db_checks');
```

Comment:
```suggestion
        $ignoreModels = Config::inst()->get(static::class, 'exclude_models_from_db_checks');
```
Only use the word 'models' instead of have 'models' and 'dataobjects'

Reply:
Done